opal/mca/btl/ofi/btl_ofi_component.c

```diff
@@ -268,8 +259,12 @@ static mca_btl_base_module_t **mca_btl_ofi_component_init (int *num_btl_modules,
             break;
     }
 
-    /* Select the provider */
-    fabric_attr.prov_name = prov_include;
+    fabric_attr.prov_name = NULL;
+    /* Select the provider - sort of.  we just take first element in list for now */
+    if (NULL != *opal_common_ofi.prov_include) {
+        include_list = opal_argv_split(*opal_common_ofi.prov_include, ',');
+        fabric_attr.prov_name = include_list[0];
+    }
```

Comment:
What happens if opal_common_ofi.prov_include is NULL? I assume that mean we can choose from any BTL?

Reply:
yes that's the way the logic currently works for both MTL and BTL.